environment.yml

```
@@ -1,11 +1,15 @@
-name: workshop
+name: image-analysis-23
 channels:
   - conda-forge
 dependencies:
+  - python=3.10
+  - pip
   - jupyterlab
   - jupytext
-  - napari
-  - pip
+  - matplotlib
+  - napari=0.4.17
+  - scikit-image=0.20.0
+  - jupyter-book=0.15.1
+  - napari-animation
   - pip:
-    - jupyter-book
-    - napari-animation
+    - napari==0.4.18rc2
```

```suggestion
    - napari==0.4.18rc2
    - scikit-image==0.21.0
```
No real reason not to use the most recent version of scikit-image just because it's from PyPI. Could we update this file if we manage to get out scikit-image 0.21 on conda-forge in the meantime? https://github.com/conda-forge/scikit-image-feedstock/pull/103 is hopefully close to merge. :pray:

---

I intend to update the instructions once both skimage 0.21 and napari 0.4.18 are out on conda-forge, hopefully early next week. But, sure, I'll add 0.21 here.